requirements.txt

```diff
@@ -1,3 +1,2 @@
-kubernetes
-PrettyTable
-urllib3
```

Comment:
The requirements.txt didn't seem to be in active used since the it wasn't referenced anywhere as far as I could tell, so I took the liberty to update the requirements.txt file and update the Dockerfile to actually install from this file. PrettyTable and urllib3 as listed in the requirements.txt at the time were probably just historic artefacts.

The Dockerfiles installed PTable as far as I could tell which would make sense as it is the more recently maintained fork of PrettyTable. I noticed that the code did reference PrettyTable but since PTable exposes a `prettytable` module it may just be that it all works fine but it would need to be tested.

There is still an `import urllib3` statement somewhere in the source but we don't reference urllib3 in the code anymore so I'm guessing it's safe to go ahead and remove every urllib3 related bit of sourcecode.